README.md

```diff
@@ -54,25 +54,27 @@ Note that the example uses the `unless` parameter to make the resource idempoten
 
 **Note:** PowerShell variables (e.g. `$_`), must be escaped in Puppet manifests either using backslashes or single quotes.
 
-Alternatively, you can put the PowerShell code for the `command`, `onlyif`, and `unless` parameters into separate templates and then invoke the template function in the resource.
+Alternatively, you can put the PowerShell code for the `command`, `onlyif`, and `unless` parameters into separate files and then invoke the file function in the resource. Templates and the `template()` function could also be used here if the PowerShell scripts need to have access to variables from Puppet.
 
 ~~~
 exec { 'rename-guest':
-  command   => template('guest/rename-guest.ps1'),
-  onlyif    => template('guest/guest-exists.ps1'),
+  command   => file('guest/rename-guest.ps1'),
+  onlyif    => file('guest/guest-exists.ps1'),
   provider  => powershell,
   logoutput => true,
 }
 ~~~
 
-Each template is a PowerShell script.
+Each file is a PowerShell script that should be in the module's `files/` folder.
```

Comment:
If you want this on the next line, you need whitespace here, otherwise markdown will render it on the same line.